libmscore/measure.cpp

```diff
@@ -2752,7 +2752,7 @@ bool Measure::isEmpty(int staffIdx) const
             strack = staffIdx * VOICES;
             etrack = strack + VOICES;
             }
-      for (Segment* s = first(SegmentType::ChordRest); s; s = s->next(SegmentType::ChordRest)) {
+      for (Segment* s = first(SegmentType::ChordRest | SegmentType::Clef); s; s = s->next(SegmentType::ChordRest | SegmentType::Clef)) {
```

Comment:
Will this affect mutlimeasure rests at all, or is this code path not hit?

Reply:
Yes, if the user deliberately places a clef on a staff he interrupts multimeasure rests. Doesn't affect other parts tho. 
If this is a problem, I can leave "isEmpty" as it is and create another function only for use just on cutaway cases ("Measure::isCutAway" or something like that)

Reply:
That much is good and expected and the same as current.  I was concerned it would make the measure containing the clef separate out from the mutlimeasure rest completely.  I think the function you are modifying here probably isn't actually called; instead there is a different function that makes the determination what constitutes empty for that purpose.

Reply:
I searched for calls to Measure::isEmpty() on all the score and, unless I missed something, it's only used for Cutaway and to Hide Staves. 
(If that's the case we might even add more element types that "unhide" the measures - but so far I only thought on the need for clefs).

Reply:
@MarcSabatella please have a look at pr #6409 when you have the time.